src/hott/04-half-adjoint-equivalences.rzk.md

```diff
@@ -768,15 +768,15 @@ have equivalent identity types by showing that equivalences are embeddings.
   ( x y : A)
   : has-section (x = y) (f x = f y) (ap A B x y f)
   :=
-    ( second (iff-ap-is-half-adjoint-equiv x y) ,
-      section-htpy-ap-is-half-adjoint-equiv x y)
+    ( second (iff-ap-is-half-adjoint-equiv x y)
+    , section-htpy-ap-is-half-adjoint-equiv x y)
 
 #def is-equiv-ap-is-half-adjoint-equiv uses (is-hae-f)
   ( x y : A)
   : is-equiv (x = y) (f x = f y) (ap A B x y f)
   :=
-    ( has-retraction-ap-is-half-adjoint-equiv x y ,
-      has-section-ap-is-half-adjoint-equiv x y)
+    ( has-retraction-ap-is-half-adjoint-equiv x y
+    , has-section-ap-is-half-adjoint-equiv x y)
```

Comment:
Oh, I forgot to mention, the convention is that after the walrus separator, code starts with the same indentation level:
```suggestion
  :=
  ( has-retraction-ap-is-half-adjoint-equiv x y
  , has-section-ap-is-half-adjoint-equiv x y)
```